### DISCUSSION TITLE
Tailwind styling not working with Twin Macro

### DISCUSSION THREAD

**Discussion by babophett** [ben-rogerson/twin.macro#827] [Q&A]
Hi,
I'm new to twin macro. I am trying to use string literals in a classname but it seems twin macro does not support literals. I just defaulted to normal tailwind styling but it seems none of the styling is working when inside twin macro elements. Any reason why this might be the case?

**Comment by ben-rogerson** [comment]
Twin just translates tailwind classes to css objects for the css-in-js library you've chosen to do the rest of the work.

What css-in-js library are you using and what code are you trying?

These two should work:

```js
const TwImportComponent = tw.div`bg-black`
const StyledImportComponent = styled.div`${bg-black}`
```
